bench/benchmarks/layout.js

```diff
@@ -11,7 +11,8 @@ const promisify = require('pify');
 const WorkerTile = require('../../src/source/worker_tile');
 const StyleLayerIndex = require('../../src/style/style_layer_index');
 const deref = require('../../src/style-spec/deref');
-const TileCoord = require('../../src/source/tile_coord');
+const OverscaledTileID = require('../../src/source/tile_id').OverscaledTileID;
```

Comment:
I think we're migrating to `const {OverscaledTileID} = require(...);` for this type of require.